spec/controllers/idv/in_person/usps_locations_controller_spec.rb

```diff
@@ -40,117 +40,119 @@
     allow(controller).to receive(:current_sp).and_return(sp)
   end
 
-  describe '#index' do
```

Comment:
Is the idea here to move all of these parameters into some bit of code that can be shared between this controller and the smaller controller you created for the Help Center?